msteams-platform/sbs-gs-blazorapp.yml

```diff
@@ -0,0 +1,387 @@
+### YamlMime:Tutorial
+title: Build your first app using Blazor
+metadata:
+  title: Hello World with Blazor
+  description: In this tutorial, you'll learn to build Hello World app with Blazor.
+  audience: Developer
+  level: Beginner
+  ms.date: 04/03/2022
+  ms.topic: interactive-tutorial
+  nextTutorialHref: ~/get-started/get-started-overview.md
+  nextTutorialTitle: Back to Get started overview
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+
+    Blazor lets you build interactive web UIs using C#, instead of JavaScript. 
+    
+    You'll create a tab app with Blazor and the latest version of Visual Studio.
+
+    :::image type="content" source="~/assets/images/teams-toolkit-v2/blazor/your-blazor-helloworld-apps.png" alt-text="Diagram showing this app has two features, tab and Bot." border="false":::
+
+    > [!NOTE]
+    > Currently, Teams Toolkit offers templates for only a tab app with Blazor. Teams Toolkit doesn't support bot and message extension capability.
```

Comment:
we now support bot in 17.2.

Reply:
> we now support bot in 17.2.

this is write for 17.2 preview 1, in which we do not have bot yet. We can release this version of doc first and then working on update it. Since there are some MVPs are waiting for this doc.